packages/app/public/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Was it missed from a previous update?

Reply:
I think this changed when `prettier` was updated to `3.y.z`.